lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs

```diff
@@ -145,9 +147,17 @@ internal sealed class REEFIMRUClient : IIMRUClient
                 .SetJobIdentifier(jobDefinition.JobName)
                 .Build();
 
-            _reefClient.SubmitAndGetDriverUrl(imruJobSubmission);
+            _httpEndPoint = _reefClient.SubmitAndGetDriverUrl(imruJobSubmission);
 
             return null;
         }
+
+        /// <summary>
+        /// DriverHttpEndPoint returned by IReefClient after job submission
+        /// </summary>
+        public IDriverHttpEndpoint DriverHttpEndpoint
```

Comment:
What if httpEndPoint hasn't got value but the user tries to access this property?